README.md

```diff
@@ -74,7 +75,7 @@ Nannou aims to provide easy, cross-platform access to the things that artists ne
     [listeners](https://doc.rust-lang.org/std/net/struct.TcpListener.html)
     via std.
 - **Device & I/O stream APIs**:
-  - [x] [Audio](https://docs.rs/nannou/0.5.2/nannou/app/struct.Audio.html).
+  - [x] [Audio](https://docs.rs/nannou/0.8.0/nannou/app/struct.Audio.html).
```

Comment:
Thanks a lot for updating all these version updates!

I wonder if there's a way we can write these links so that we con't have to change these version numbers every time we update :thinking:

Reply:
ok no problem. If you want to *always* pin the latest version of the documentation, we can simply pin that with `latest` in the path ;-)